connectors/rocketmq-connect-email/src/main/java/org/apache/rocketmq/connect/email/sink/EmailSinkTask.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.rocketmq.connect.email.sink;
+
+import java.util.List;
+
+import org.apache.rocketmq.connect.email.config.EmailSinkConfig;
+import org.apache.rocketmq.connect.email.helper.EmailHelperClient;
+
+import io.openmessaging.KeyValue;
+import io.openmessaging.connector.api.component.task.sink.SinkTask;
+import io.openmessaging.connector.api.data.ConnectRecord;
+import io.openmessaging.connector.api.errors.ConnectException;
+
+public class EmailSinkTask extends SinkTask {
+
+    private EmailSinkConfig config;
+    private EmailHelperClient emailClient;
+
+    public EmailSinkConfig getConfig() {
+        return config;
+    }
+
+    public EmailHelperClient getEmailClient() {
+        return emailClient;
+    }
+
+    @Override
+    public void start(KeyValue keyValue) {
+        this.config = new EmailSinkConfig();
+        this.config.load(keyValue);
+        this.emailClient = new EmailHelperClient(this.config);
+    }
+
+    @Override
+    public void stop() {
+        this.emailClient = null;
+    }
+
+    @Override
+    public void put(List<ConnectRecord> sinkRecords) throws ConnectException {
+        if (sinkRecords == null || sinkRecords.isEmpty()) {
+            return;
+        }
+
+        for (ConnectRecord record : sinkRecords) {
+            this.emailClient.send(record);
```

Comment:
maybe consider batch sending emails?  Too much emails will be sent, and attachment files and content may contain little information